From: Sam Hartman <hartmans@debian.org>
Date: Mon, 8 Jul 2019 21:54:25 -0400
Subject: Some more des test failures

Fix some more des related test failures.


I have not actually gotten make check working but I've convinced
myself the remaining failures are test failures not code failures.
This will all clean up after the 1.18 release and we don't currently
run make check during the build anyway.
---
 src/lib/krb5/krb/t_get_etype_info.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/lib/krb5/krb/t_get_etype_info.py b/src/lib/krb5/krb/t_get_etype_info.py
index 7c400be..16eaf1f 100644
--- a/src/lib/krb5/krb/t_get_etype_info.py
+++ b/src/lib/krb5/krb/t_get_etype_info.py
@@ -9,8 +9,8 @@ realm.run([kadminl, 'ank', '-nokey', '+preauth', 'pnokey'])
 realm.run([kadminl, 'ank', '-e', 'aes256-cts:special', '-pw', 'pw', 'exp'])
 realm.run([kadminl, 'ank', '-e', 'aes256-cts:special', '-pw', 'pw', '+preauth',
            'pexp'])
-realm.run([kadminl, 'ank', '-e', 'des-cbc-crc:afs3', '-pw', 'pw', 'afs'])
-realm.run([kadminl, 'ank', '-e', 'des-cbc-crc:afs3', '-pw', 'pw', '+preauth',
+realm.run([kadminl, 'ank', '-e', 'des3-cbc-sha1:afs3', '-pw', 'pw', 'afs'])
+realm.run([kadminl, 'ank', '-e', 'des3-cbc-sha1:afs3', '-pw', 'pw', '+preauth',
            'pafs'])
 
 # Extract the explicit salt values from the database.
@@ -56,7 +56,7 @@ realm.run(['./t_get_etype_info', 'exp'],
 realm.run(['./t_get_etype_info', 'pexp'],
           expected_msg='etype: aes256-cts\nsalt: ' + pexp_salt + '\n')
 
-msg = 'etype: des-cbc-crc\nsalt: KRBTEST.COM\ns2kparams: 01\n'
+msg = 'etype: des3-cbc-sha1\nsalt: KRBTEST.COM'
 realm.run(['./t_get_etype_info', 'afs'], expected_msg=msg)
 realm.run(['./t_get_etype_info', 'pafs'], expected_msg=msg)
 
